AIM/AIM/UI/Views/TrackListViewController.m

```diff
@@ -0,0 +1,109 @@
+//
+//  TrackListViewController.m
+//  AIM
+//
+//  Created by Oskar Szydlowski on 05.03.2018.
+//  Copyright © 2018 Oskar Szydlowski. All rights reserved.
+//
+
+#import "TrackListViewController.h"
+
+#import "TrackCell.h"
+
+@interface TrackListViewController () {
```

Comment:
ARC not used

Reply:
Well, it turned out I was wrong, we can write iVars that way and we still use arc. My knowledge about Obj-C is a bit rusty as well :P